discover/home/core-features/human-check.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Human Check"
+description: "Unlock exclusive rewards with a quick verification that confirms you're a real person."
+---
+
+**What is Human Check?**
+
+Human Check is a quick liveness verification that uses your camera to confirm you're a real Sophon member, not a bot or automated system. It protects our most valuable rewards in Sophon\+ and keeps the ecosystem fair for everyone.
```

Comment:
"real Sophon member" might make readers confused. Gives an impression that we are validating employees for example. Would keep it simple "real person"